### ISSUE TITLE
Help Restore WhatsApp View Once Bypass After Latest Security Update

### ISSUE THREAD

**Issue by tharinduxd0** [tharinduxd0/whatsapp-view-once-bypass-chrome-extension#2]
### **Important Update**

Due to a recent security update released by WhatsApp in OCT 2024, the current version of the extension is no longer fully functional. This update has affected the extension’s ability to bypass the "View Once" feature.

**I am actively working on an updated version to restore the full functionality. However, I would like to invite the community to help fix this issue. If anyone can contribute by reviewing the code, rewriting parts of the program, or offering suggestions, your assistance would be greatly appreciated.**

Feel free to clone the repository, experiment with the code, and submit pull requests if you find solutions. Let’s work together to get this extension back up and running.

Thank you for your patience and understanding. 
Stay tuned for further updates!

#1

**Comment by juliokyoma** [comment]
Well if I log out and Log in The Extension works, just once. I don't know if it helps

**Comment by eiidrovo** [comment]
> ### **Important Update**
> Due to a recent security update released by WhatsApp in OCT 2024, the current version of the extension is no longer fully functional. This update has affected the extension’s ability to bypass the "View Once" feature.
> 
> **I am actively working on an updated version to restore the full functionality. However, I would like to invite the community to help fix this issue. If anyone can contribute by reviewing the code, rewriting parts of the program, or offering suggestions, your assistance would be greatly appreciated.**
> 
> Feel free to clone the repository, experiment with the code, and submit pull requests if you find solutions. Let’s work together to get this extension back up and running.
> 
> Thank you for your patience and understanding. Stay tuned for further updates!
> 
> #1

Any advance?

**Comment by S3idSensei** [comment]
Yeah, all extensions are like that now, you need to re-login to see view once messages that are already sent, if you get a new view once message, you need to re-login to see it.

**Comment by TareqAbAmawi** [comment]
Any updates now view once photos stopped working !

**Comment by TareqAbAmawi** [comment]
Any updates now view once photos stopped working !

**Comment by Nvz14** [comment]
I guess this is still not working?

**Comment by sp201004** [comment]
Any update regarding WhatsApp View Once Bypass?

**Comment by sp201004** [comment]
which extension?

**Comment by codingshukla** [comment]
whatsapp is not allowing to open the view once photos on the web

**Comment by glenhassan45-code** [comment]
😂 😂
